src/app/screens/Publishers/index.tsx

```diff
@@ -90,21 +103,107 @@ function Publishers() {
 
   return (
     <Container>
-      <h2 className="mt-12 mb-2 text-2xl font-bold dark:text-white">
-        {t("used.title")}
-      </h2>
+      {(publishers.length > 0 || !hasTips) && (
+        <>
+          <h2 className="mt-12 mb-2 text-2xl font-bold dark:text-white">
+            {t("used.title")}
+          </h2>
+          <p className="mb-6 text-gray-500 dark:text-neutral-500">
+            {t("used.description")}
+          </p>
 
-      <p className="mb-6 text-gray-500 dark:text-neutral-500">
-        {t("used.description")}
-      </p>
-
-      {publishers.length > 0 ? (
-        <PublishersTable
-          publishers={publishers}
-          navigateToPublisher={navigateToPublisher}
-        />
-      ) : (
-        <p className="dark:text-white"> {t("used.no_info")}</p>
+          {publishers.length > 0 ? (
+            <PublishersTable
+              publishers={publishers}
+              navigateToPublisher={navigateToPublisher}
+            />
+          ) : (
+            <p className="dark:text-white"> {t("used.no_info")}</p>
+          )}
+        </>
+      )}
+      {publishers.length === 0 && hasTips && (
+        <>
+          <h2 className="mt-12 mb-2 text-2xl font-bold dark:text-white">
+            {t("tips.title")}
+          </h2>
+          <p className="mb-6 text-gray-500 dark:text-neutral-500">
+            {t("tips.description")}
+          </p>
+          <div className="grid gap-4 sm:grid-cols-1 lg:grid-cols-2">
```

Comment:
It could be good to use an array and map (some things like handleClose() would not need to be duplicated)

Reply:
i thought about that but there is a lot good bit of differences in the card layouts such as showing a numbered list, different buttons in the footer, etc. I can do it in a single loop but there will likely be added complexity to determine things like which buttons to show inside the loop. Although the non-loop approach is more verbose it felt less complex and more adaptable to future changes.  Happy to do it either way.

Reply:
I kinda held myself back to ask for some loop and json-file because I thought it's really not necessary yet and who knows where this is going. Further thoughts where things like "markdown" and "oh, now we're building a CMS" :D